admin/client.go

```diff
@@ -201,13 +201,14 @@ func (c *Client) ListChangeSummaries(
 		return nil, err
 	}
 
-	lastSeq := changes[len(changes)-1].ID().ServerSeq()
-	from, _ := types.GetChangesRange(types.Paging[uint64]{
-		Offset:    previousSeq,
-		PageSize:  int(pageSize),
-		IsForward: isForward,
-	}, lastSeq)
-	seq := from - 1
+	// lastSeq := changes[len(changes)-1].ID().ServerSeq()
+	// from, _ := types.GetChangesRange(types.Paging[uint64]{
+	// 	Offset:    previousSeq,
+	// 	PageSize:  int(pageSize),
+	// 	IsForward: isForward,
+	// }, lastSeq)
+	// seq := from - 1
```

Comment:
The above ListChanges function is already calling the GetChangesRange function, so I omitted it.